Source/Core/DiscIO/NANDContentLoader.cpp

```diff
@@ -495,6 +487,7 @@ u64 CNANDContentManager::Install_WiiWAD(const std::string& filename)
 
   tmd_file.WriteBytes(content_loader.GetTMDHeader(), CNANDContentLoader::TMD_HEADER_SIZE);
 
+  CSharedContent shared_content{Common::FromWhichRoot::FROM_CONFIGURED_ROOT};
```

Comment:
What I was thinking of was to create the object inside the `if` block:

```
if (content.m_Type & 0x8000)  // shared
{
  CSharedContent shared_content{Common::FromWhichRoot::FROM_CONFIGURED_ROOT};
  app_filename = shared_content.AddSharedContent(content.m_SHA1Hash);
}
else
{
  app_filename = StringFromFormat("%s%08x.app", content_path.c_str(), content.m_ContentID);
}
```

...but now I realize that it actually is a terrible idea because it's in a loop. So please ignore my suggestion.